concordium-consensus/src/Concordium/Kontrol/Bakers.hs

```diff
@@ -196,20 +197,19 @@ computeBakerStakesAndCapital poolParams activeBakers passiveDelegators = BakerSt
               capLimit
             ]
         )
-    bakerStakes = zipWith makeBakerStake activeBakers poolCapitals
+    activeBakers' = [abi | abi@ActiveBakerInfo{..} <- activeBakers, not activeBakerIsSuspended]
+    poolCapitals' = poolCapital <$> activeBakers'
```

Comment:
*Suggestion (non-blocking):* consider `filteredActiveBakers` and `filteredPoolCapitals` rather than `activeBakers'` and `poolCapitals'`.